db/sql/sql.go

```diff
@@ -984,10 +1120,56 @@ func (tx *Transaction) LockListContext(ctx context.Context, s *schema.Schema, fi
 	return tx.executeSelect(ctx, sc, sql, args)
 }
 
+// LockList locks resources in the db
+func (tx *CachedTransaction) LockListContext(ctx context.Context, s *schema.Schema, filter transaction.Filter, options *transaction.ViewOptions, pg *pagination.Paginator, lockPolicy schema.LockPolicy) (list []*schema.Resource, total uint64, err error) {
+
+	policyJoin := shouldJoin(lockPolicy)
+
+	sc := &selectContext{
+		schema:    s,
+		filter:    filter,
+		join:      policyJoin,
+		paginator: pg,
+	}
+	if options != nil {
+		sc.fields = normFields(options.Fields, s)
+		sc.join = policyJoin && options.Details
+	}
+
+	sql, args, err := buildSelect(sc)
+	if err != nil {
+		return nil, 0, err
+	}
+
```

Comment:
```	
if tx.db.sqlType == "mysql" {
    sql += " FOR UPDATE"
}
```
is missing

Reply:
results will be the same, even if we skip "for update".